tree-sitter-stack-graphs/src/loader.rs

```diff
@@ -75,6 +76,15 @@ impl LanguageConfiguration {
         let mut builtins = StackGraph::new();
         if let Some((builtins_path, builtins_source)) = builtins_source {
             let mut builtins_globals = Variables::new();
+
+            let builtins_path_var = Path::new("<builtins>");
+            builtins_globals
+                .add(
+                    FILE_PATH_VAR.into(),
+                    builtins_path_var.to_str().unwrap().into(),
+                )
+                .expect("failed to add file path variable");
```

Comment:
Since we're not using this anywhere else, I'd just inline `builtins_path_var here.

```suggestion
            builtins_globals
                .add(
                    FILE_PATH_VAR.into(),
                    "<builtins>".into(),
                )
                .expect("failed to add file path variable");
```